### DISCUSSION TITLE
"new device" gives 404

### DISCUSSION THREAD

**Discussion by dimpase** [osmandapp/OsmAnd#14344] [Maps Related]
I recently got a new phone and would like to transfer my OsmAnd subscription there. 

However "new device" link just doesn't work.
What should I do?

**Comment by dmpr0** [comment]
You need to use the same Google account, if the subscription doesn't work, try to tap on "Restore purchase".

**Reply by dimpase** [reply]
It doesn't work either way.
Not 100% sure what "the same Google account" means. I've got a new phone, [Fairphone 4, from /e/ foundation](https://esolutions.shop/), a "de-googled" Android phone.
I am able to access Google services such as gmail just fine, but it's not a full set of Google service.

On 22.09.2021 I bought "Maps+, One-time payment" - not sure whether this kind of subscription is still available for purchase. It was around 30 euro if I recall right, (can't find the receipt right now), and meant to be a kind of life-time paid account. OsmAnd appears to work fine on the new phone, but I am not able to transfer this purhase.

**Reply by dmpr0** [reply]
If you download OsmAnd from Google Play, all purchases will be linked to Google Account that was primary no device."Restore purchase" connects to OsmAnd servers and Publisher information (Google Play, Amazon, Huawei, Apple) to get confirmation about payments.
Here is the correct "New device" link https://docs.osmand.net/docs/user/purchases/#new-device--new-account

**Reply by dimpase** [reply]
I've installed OsmAnd from /e/ foudation mirror, which is basically [cleanapk.org](https://info.cleanapk.org/), see their [FAQ](https://doc.e.foundation/apps#where-do-the-apps-in-e-installer-application-come-from). I know it's version 4.1.11 (421), updated 2012/12/16, and package name is `net.osmand`.

I can't use Google Play Store - but this appears to be the same version I have on the old phone.

If I need to install something else, is there an apk file I can get somewhere?

---
Here are more details  on the purchase (via pay.google.com): 
```
Maps+ (OsmAnd — Offline Maps, Travel & Navigation)
                £24.99

DATE TIME
Wednesday, September 22, 2021 5:58 AM

TRANSACTION ID   GPA.3320-5016-9556-37154

PURCHASED FROM
Osmand
Logger 41
1186RM Amstelveen
Netherlands
```

Probably the problem is that the new phone does not support Google Pay (for the same reason it does not support Google Play Store)

**Reply by dimpase** [reply]
It won't be a problem for me to buy a new subscription - would you be willing to provide a refund for the old one then?

**Reply by dimpase** [reply]
Oops, no, sorry, I actually can't seem to be able to the in-app purchase from OsmAnd on the new device. 

I get "Cannot launch full version purchase!" if I select "One-time payment - EUR 24,99" and tap "Complete purchase".

**Reply by dmpr0** [reply]
Purchases work thru Google Play. You can get a refund from Google Play or contact our support at support@osmand.net.
